src/plugins/unified_field_list/kibana.jsonc

```diff
@@ -5,7 +5,7 @@
   "description": "Contains functionality for the field list which can be integrated into apps",
   "plugin": {
     "id": "unifiedFieldList",
-    "server": true,
+    "server": false,
```

Comment:
Removed the server code. One step closer to converting from a plugin to a package!

Reply:
Great PR removing legacy and providing documentation that can be tested! Just a question, it migrating to packages would have been possible before, but it would need 2 different packages for public and server code, much more effort right?

Reply:
@kertal Yes, and a server package would consist of only routes for tests (if it's even possible to test package routes with functional API tests)